### PR TITLE
fix #83104 dailyuploads.net

### LINKED ISSUE #83104: dailyuploads.net
### Issue URL (Ads)
 [https://dailyuploads.net/c73plnt2x6uj](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdailyuploads.net%2Fc73plnt2x6uj)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6.1
Browser: | Edge 90.0.818.56 (Chromium-based)
AdGuard driver: | WFP
Stealth mode: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental<br/><br/><b>Language-specific:</b><br/> Icelandic ABP List, <br/>Dandelion Sprout's Nordic Filters, <br/>AdGuard German, <br/>AdGuard Japanese
Userscripts: | https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js,<br>https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
Other extensions used: | no

### PR DESCRIPTION
**Pull request by wout-codes**
#83104

`dailyuploads.net` was removed from the `zfgloaded` section.

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -148,9 +148,9 @@ watchseries-online.nl,btsone.cc#%#Object.defineProperty(window,'_drp', { get:fun
...
...ntsmd.eu.org,tous-sports.tv,toussports.info,tubebg.com,tubecomplet.com,tudotv.tv,tudotvonline.com,tugaanimado.net,tumejortorrent.com,tusnovelas.net,tv-onlinehd.com,twojetv.ws,uii.io,upload.ac,upload4earn.org,uploadbank.com,uploadmp4.com,upzone.cc,urgetofap.com,urgrove.net,userscloud.com,ustreamix.com,vezi-online.com,vidbob.com,vidcloud.icu,videokhoj.co,vidfile.net,vidload.co,vidlox.me,vidlox.tv,vidnode.net,vidoevo.com,vidstreaming.io,vidtomp3.info,vimeotomp3.com,viralitytoday.com,vizhe.xyz,vojkudee.net,vstarmusiq.com,watch-online.biz,watch-series.cc,watch-series.co,watch32.is,watch5s.me,watcharcheronline.com,watchcartoonsonline.eu,watchfilmy.com,watchmovies.pk,watchncum.biz,watchncum.com,watchonline.me,watchonlinemovies.com.pk,watchparksandrecreation.net,watchrecentmovies.co,watchseries.unblockme.xyz,watchserieshd.cc,watchserieshd.io,watchshow.stream,wicr.me,xeimg.yt,xkeezmovies.com,xmovies8.io,xmovies8.net,xmoviesforyou.com,xmoviesforyou1.com,xrares.com,xxxstreams.eu,xxxwebdlxxx.org,yakaracolombia.com,yalla-shoot.com,yeane.org,ymovies.tv,yourporn.sexy,yt2mp3s.me,yttomp3.org,yuyalay.com,zedstream.com,zinkhd.co$$script[tag-content="zfgloaded"][min-length="30000"][max-length="80000"]
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:151]: What's changed here? Did you remove `dailyuploads.net`?

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -148,9 +148,9 @@ watchseries-online.nl,btsone.cc#%#Object.defineProperty(window,'_drp', { get:fun
...
...tlocura.com,torrentmegafilme.com,torrentrapid.com,torrents9.org,torrentsmd.eu.org,tous-sports.tv,toussports.info,tubebg.com,tubecomplet.com,tudotv.tv,tudotvonline.com,tugaanimado.net,tumejortorrent.com,tusnovelas.net,tv-onlinehd.com,twojetv.ws,uii.io,upload.ac,upload4earn.org,uploadbank.com,uploadmp4.com,upzone.cc,urgetofap.com,urgrove.net,userscloud.com,ustreamix.com,vezi-online.com,vidcloud.icu,videokhoj.co,vidfile.net,vidload.co,vidlox.tv,vidnode.net,vidoevo.com,vidstreaming.io,vidtomp3.info,vimeotomp3.com,viralitytoday.com,vizhe.xyz,vojkudee.net,vstarmusiq.com,watch-online.biz,watch-series.co,watch32.is,watch5s.me,watcharcheronline.com,watchfilmy.com,watchmovies.pk,watchncum.biz,watchncum.com,watchonline.me,watchonlinemovies.com.pk,watchparksandrecreation.net,watchrecentmovies.co,watchseries.unblockme.xyz,watchserieshd.cc,watchshow.stream,wicr.me,xeimg.yt,xkeezmovies.com,xmovies8.io,xmovies8.net,xmoviesforyou.com,xmoviesforyou1.com,xrares.com,xxxstreams.eu,xxxwebdlxxx.org,yakaracolombia.com,yalla-shoot.com,yeane.org,ymovies.tv,yourporn.sexy,yt2mp3s.me,yttomp3.org,yuyalay.com,zedstream.com,zinkhd.co#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "zfgloaded")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:153]: The same question.

**Comment by Yuki2718** [comment]
Not sure if "delete window" is preferred, as either will match.

File: EnglishFilter/sections/general_extensions.txt (hunk 1 above)
Comment (Yuki2718) [reply EnglishFilter/sections/general_extensions.txt:151]: Looks like so and I also don't see zfgloaded.

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:151]: > Looks like so and I also don't see zfgloaded.

At first look, I see the same thing. But for such commits, it is preferable to write what exactly has changed.

Comment (Yuki2718) [reply EnglishFilter/sections/general_extensions.txt:151]: Oh, okay I remember this as well.

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:153]: What's wrong? I removed this domain.

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:153]: ```dailyuploads.net``` has been removed from this rule.

Comment (Yuki2718) [reply EnglishFilter/sections/general_extensions.txt:153]: @wout-codes I believe nothing wrong, apparently he just wanted comment describing what you did - it's hard to find change in such a big filter.

Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:153]: > What's wrong? I removed this domain.

Nothing, but you shoud write about it in the description when you work with such rules https://github.com/AdguardTeam/AdguardFilters/pull/83106#discussion_r631683043

Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:153]: Ok

**Review by zloyden — APPROVED** [review]

**Pull request merged**